invenio_vocabularies/contrib/affiliations/datastreams.py

```diff
@@ -96,15 +99,24 @@ def _entry_id(self, entry):
         """Get the id from an entry."""
         return entry["id"]
 
-    def write(self, stream_entry, *args, **kwargs):
-        """Writes the input entry using a given service."""
-        entry = stream_entry.entry
-
-        return super().write(stream_entry, *args, **kwargs)
-
-    def write_many(self, stream_entries, *args, **kwargs):
-        """Writes the input entries using a given service."""
-        return super().write_many(stream_entries, *args, **kwargs)
```

Comment:
`write` and `write_many` were not needed since the validation logic was moved to the transformer and these methods were simply calling their parent class.